graph/case_similarity.py

```diff
@@ -28,6 +29,21 @@ def most_similar_to_group(self, opinion_ids: set) -> Dict[str, float]:
                 / len(opinion_neighbor_sets)
         return similarity_value_dict
 
+    def internal_similarity(self, opinion_ids: set) -> Dict[frozenset, float]:
```

Comment:
This function returns an `nx.Graph`, why is the return type `Dict[frozenset, float]`?

Reply:
because it used to be a Dict lmao :grimacing: good catch